components/writers/src/main/java/org/datacleaner/beans/writers/UpdateDataHubAnalyzer.java

```diff
@@ -0,0 +1,570 @@
+import org.apache.metamodel.create.TableCreationBuilder;
+import org.apache.metamodel.csv.CsvDataContext;
+import org.apache.metamodel.insert.RowInsertionBuilder;
+import org.apache.metamodel.query.FilterItem;
+import org.apache.metamodel.query.OperatorType;
+import org.apache.metamodel.query.SelectItem;
+import org.apache.metamodel.schema.Column;
+import org.apache.metamodel.schema.ColumnType;
+import org.apache.metamodel.schema.Schema;
+import org.apache.metamodel.schema.Table;
+import org.apache.metamodel.update.RowUpdationBuilder;
+import org.apache.metamodel.util.Action;
+import org.apache.metamodel.util.FileHelper;
+import org.apache.metamodel.util.Resource;
+import org.datacleaner.api.Analyzer;
+import org.datacleaner.api.Categorized;
+import org.datacleaner.api.ColumnProperty;
+import org.datacleaner.api.ComponentContext;
+import org.datacleaner.api.Concurrent;
+import org.datacleaner.api.Configured;
+import org.datacleaner.api.Description;
+import org.datacleaner.api.ExecutionLogMessage;
+import org.datacleaner.api.FileProperty;
+import org.datacleaner.api.FileProperty.FileAccessMode;
+import org.datacleaner.api.HasLabelAdvice;
+import org.datacleaner.api.Initialize;
+import org.datacleaner.api.InputColumn;
+import org.datacleaner.api.InputRow;
+import org.datacleaner.api.MappedProperty;
+import org.datacleaner.api.Provided;
+import org.datacleaner.api.SchemaProperty;
+import org.datacleaner.api.TableProperty;
+import org.datacleaner.api.Validate;
+import org.datacleaner.components.categories.WriteSuperCategory;
+import org.datacleaner.components.convert.ConvertToBooleanTransformer;
+import org.datacleaner.components.convert.ConvertToNumberTransformer;
+import org.datacleaner.connection.CsvDatastore;
+import org.datacleaner.connection.FileDatastore;
+import org.datacleaner.connection.SchemaNavigator;
+import org.datacleaner.connection.UpdateableDatastore;
+import org.datacleaner.connection.UpdateableDatastoreConnection;
+import org.datacleaner.data.MetaModelInputColumn;
+import org.datacleaner.descriptors.FilterDescriptor;
+import org.datacleaner.descriptors.TransformerDescriptor;
+import org.datacleaner.desktop.api.PrecedingComponentConsumer;
+import org.datacleaner.job.builder.AnalysisJobBuilder;
+import org.datacleaner.util.WriteBuffer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Named("Update DataHub")
+@Description("Update Golden Records in the DataHub. This component allows you to update fields defined in the golden records metadata. The component always updates a golden record based"
+        + "\non its golden record id."
+        + "\nThe 'values to update' property lets you specify which values must be changed. This component can not update fields of the metadata, like traffic lights or severities or track codes.")
+@Categorized(superCategory = WriteSuperCategory.class)
+@Concurrent(true)
+public class UpdateDataHubAnalyzer implements Analyzer<WriteDataResult>, Action<Iterable<Object[]>>, HasLabelAdvice,
```

Comment:
But I mean in general ... is there anything different in the code at all? If not, then we could just use the existing UpdateTableAnalyzer?